### PR TITLE
Bxmsdoc 5864 master

### PR DESCRIPTION
**Pull request by mramendi**

### CODE REVIEW

File: doc-content/enterprise-only/openshift/operator-deploy-pim-proc.adoc [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+. In the *Database type* field, select the database that the PIM service must use. The following values are available:
+** `mysql`: A MySQL server, created in a separate pod.
+** `postgresql`: A PostgreSQL server, created in a separate pod. Use this setting unless you have a specific reason to use any other setting.
+** `h2`: A built-in `h2` database engine that does not require a separate pod.
+** `external`: An external database server. 
+. If you selected any database except `external`, a Persistent Volume Claim will be created to store the database. Optionally, set configuration parameters of the persistent volume to create:
+** In the *Size* field, enter the size of the persistence volume
+** In the *StorageClass name* field, enter the storage class name for the persistent volume
+. If you selected the `external` database and you want to use any database server except PostgreSQL or MySQL, you must provide a {KIE_SERVER} extension image with the database server driver according to instructions in <<externaldb-build-proc>>. The PIM service can use the {KIE_SERVER} extension image. To configure the PIM service to use this extension image, make the following changes:
```
Comment (ruromero) [inline doc-content/enterprise-only/openshift/operator-deploy-pim-proc.adoc:22]: The kie-server extension images are not related to PIM images. PIM images are built differently. It's enough to include in the image the driver somewhere accessible and add it to the classpath using  the JBOSS_KIE_EXTRA_CLASSPATH env var.

File: doc-content/enterprise-only/openshift/operator-deploy-pim-proc.adoc [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+.. Select the *Enable extension image stream* box.
```
Comment (ruromero) [inline doc-content/enterprise-only/openshift/operator-deploy-pim-proc.adoc:23]: That is not part of the PIM deployment

File: doc-content/enterprise-only/openshift/operator-deploy-pim-proc.adoc (hunk 1 above)
Comment (mramendi) [reply doc-content/enterprise-only/openshift/operator-deploy-pim-proc.adoc:22]: Unfortunately, we can't make it accessible *without* a custom image (or some configmap approach). I will therefore change the doc to list only MySQL/PostgreSQL/MariaDB as possible DB servers for PIM, and create an RHPAM issue about absence of a way to provide an external driver. I will create a review task for you once that is done.

File: doc-content/enterprise-only/openshift/operator-deploy-pim-proc.adoc (hunk 2 above)
Comment (mramendi) [reply doc-content/enterprise-only/openshift/operator-deploy-pim-proc.adoc:23]: But it is in the UI for PIM. I will remove it from the doc but as mentioned above I'll make an RHPAM task about this.

Comment (mramendi) [reply doc-content/enterprise-only/openshift/operator-deploy-pim-proc.adoc:23]: My mistake, sorry. It is not in the UI.

Comment (mramendi) [reply doc-content/enterprise-only/openshift/operator-deploy-pim-proc.adoc:23]: The review request will be coming after I sort out dialects, as these need to come in here too.

File: doc-content/enterprise-only/openshift/operator-deploy-pim-proc.adoc [hunk 3]
```diff
@@ -0,0 +1,50 @@
+[id='operator-deploy-pim-proc']
+= Setting Process Instance Migration configuration for the environment
+
+You can use the operator to deploy the Process Instance Migration (PIM) service. You can use the PIM service to define the migration between two different process definitions, known as a migration plan. You can then apply the migration plan to the running process instance in a specific KIE Server.
```
Comment (ruromero) [inline doc-content/enterprise-only/openshift/operator-deploy-pim-proc.adoc:4]:
```suggestion
You can use the operator to deploy the Process Instance Migration (PIM) service. You can use the PIM service to define the migration between two different process definitions, known as a migration plan. You can then apply the migration plan to the running process instances in a specific KIE Server. 
```

File: doc-content/enterprise-only/openshift/operator-deploy-kieserver-proc.adoc [hunk 4]
```diff
@@ -54,12 +53,14 @@ complete the following additional steps:
...
 ifdef::PAM[]
-. Select the database that the {KIE_SERVER} must use. The following values are available:
+. In the *Database type* field, select the database that the {KIE_SERVER} must use. The following values are available:
 ** `mysql`: A MySQL server, created in a separate pod.
 ** `postgresql`: A PostgreSQL server, created in a separate pod. Use this setting unless you have a specific reason to use any other setting.
 ** `h2`: A built-in `h2` database engine that does not require a separate pod. Do not scale the {KIE_SERVER} pod if you use this setting.
 ** `external`: An external database server. 
-. If you selected any database except `external`, a Persistent Volume Claim will be created to store the database. Optionally, in the *Size* field, enter the size of the persistence volume to create.
+. If you selected any database except `external`, a Persistent Volume Claim will be created to store the database. Optionally, set configuration parameters of the persistent volume to create:
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/operator-deploy-kieserver-proc.adoc:61]: Suggestion - move "optionally" and resolve sentence fragment "to create":

Optionally, if you selected any database except `external`, a Persistent Volume Claim is created to store the database and you must set the configuration parameters for the persistent volume:

File: doc-content/enterprise-only/openshift/operator-deploy-kieserver-proc.adoc [hunk 5]
```diff
@@ -54,12 +53,14 @@ complete the following additional steps:
...
+** In the *Size* field, enter the size of the persistence volume
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/operator-deploy-kieserver-proc.adoc:62]: Suggestion - Add periods to each bullet point.

**Pull request merged**